static/css/opening.css

```diff
@@ -0,0 +1,22 @@
+/* html {
```

Comment:
This file can be deleted from this branch. A backlog ticket was added and this feature will be worked on later.

Reply:
This one is deleted.